### ISSUE TITLE
Consolidate tags

### ISSUE THREAD

**Issue by chrisnatali** [SEL-Columbia/SEL-Columbia.github.io#89]
We don't need to associate posts with every tag we can think of...that's why we have search (via ```term [term]* site:sel.columbia.edu```).  I say we limit it to max 10 tags or less.  Otherwise, posts repeat too frequently on the tags page.  

Example roll ups:
pv -> solar
data-driven planning -> data-collection
etc

**Comment by vr2262** [comment]
I don't know that this is really a problem... Tags tends to be pretty loose and I don't see the harm in having a large number of tags. If anything, a greater number of tags makes it easier for people to find things on the site.

**Comment by chrisnatali** [comment]
I see 2 harms:
- Loss of meaning...e.g. if we try and stick to the term "Photo-Voltaic", but sometimes posts get tagged as "PV" instead, then the meaning of 1 or the other gets watered down.
- Since we're listing all the posts associated with each tag on the tags page, when there's "a lot" of redundancy among the meaning of tags, we'll have a lot of redundant posts. 

It's easy enough to manage tags when it's a single person's blog.  In our case, when there are many contributors tagging things without any consistency, they can quickly get out of hand.  

So...maybe we don't need to roll everything up into 10 tags, but we should at least merge those with the same meaning so that we don't propagate the confusion.

**Comment by vr2262** [comment]
Agreed on point 1. It's not hard to consolidate the tags, even after the fact.
On point 2... eh. Who cares about redundant posts on the tags page?
